subsys/nrf_security/src/drivers/cracen/Kconfig

```diff
@@ -57,6 +57,14 @@ config CRACEN_IKG_SEED_KMU_SLOT
 	  This defines the KMU slot (along with the two following it) to which
 	  the IKG seed is provisioned and that is pushed when loading the IKG seed.
 
+config CRACEN_NO_INTERRUPTS
```

Comment:
Not sure if this should be wrapped in a "PSA_WANT" config instead? Also likely better to have a more descriptive name once there is an actual idea on what we want to call this